blackbelt/git.py

```diff
@@ -1,5 +1,5 @@
 import re
-from subprocess import check_output
```

Comment:
I still don't know why people hate this :)

Reply:
I found I have to change it because of the testing. When I patched the `check_output` and kept import, it was calling an original function rather then a mock.

Reply:
...right, true

Reply:
Never met anyone hating this :)